english/your-dog/your-dog.md

```diff
@@ -1,3 +1,7 @@
-Then you wake up! Or did you really? But your dog is licking you in the face. Lord, that dog breath!
+Then you wake up! Or did you really? But your dog is licking you in the face. Lord, that dog breath!  Almost as fragrent as when he comes in from the rain.
 
-Then your dog speaks to you and says ["follow me ..."](../play-with-dog/dog.md)
\ No newline at end of file
+Then your dog speaks to you and says ["follow me ..."](../play-with-dog/dog.md)
+
+Or is it a dream?  Dogs don't speak English afterall.  You have a couple of choices.  You can ["follow him"](../play-with-dog.md),
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.